toolkit/Makefile

```diff
@@ -64,7 +64,7 @@ PROJECT_ROOT     ?= $(realpath $(toolkit_root)/..)
 BUILD_DIR        ?= $(PROJECT_ROOT)/build
 OUT_DIR          ?= $(PROJECT_ROOT)/out
 SPECS_DIR        ?= $(PROJECT_ROOT)/SPECS
-CCACHE_DIR       ?= $(PROJECT_ROOT)/ccache
+CCACHE_DIR       ?= /home/$(SUDO_USER)/mariner-ccache
```

Comment:
How is `$(SUDO_USER)` defined?

Reply:
SUDO_USER is set to the login name of the user that invoked the sudo command. https://www.sudo.ws/docs/man/sudo.man/#SUDO_USER